tests/test-target-zip-size.sh

```
@@ -0,0 +1,160 @@
+        print_error "$scenario: size FAILED (actual=${actual_bytes}, target=${target_bytes}, deviation=${deviation} > tolerance=${tolerance})"
+        FAILED=$((FAILED + 1))
+    fi
+}
+
+assert_padding_spread() {
+    local zip_file="$1"
+    local target_bytes="$2"
+    local file_count="$3"
+    local scenario="$4"
+
+    local max_allowed_per_file=$((target_bytes / file_count * 10))
```

Shellcheck (0.11.0)

[info] Increase precision by replacing a/b*c with a*c/b.

(SC2017)